TSG.md

```diff
@@ -0,0 +1,207 @@
+# Common Mount Problems
+**Logging**
+
+Please ensure logging is turned on DEBUG mode when trying to reproduce an issue.
+This can help in many instances understand what the underlying issue is.
+
+A useful setting in your configuration file to utilize when debugging is `sdk-trace: true` under the azstorager component. This will log all outgoing REST calls.
+
+**1. Error: fusermount: failed to open /etc/fuse.conf: Permission denied**
+
+Only the users that are part of the group fuse, and the root user can run fusermount command. In order to mitigate this add your user to the fuse group.
+
+```sudo addgroup <user> fuse```
+
+**2. failed to mount : failed to authenticate credentials for azstorage**
+There might be something wrong about the storage config, please double check the storage account name, account key and container/filesystem name. errno = 1**
```

Comment:
We need to mention about how to handle proxy envrionments, as auth may fail if proxy config is not provided.